tests/python/direct/conftest.py

```diff
@@ -12,6 +12,19 @@
 from python.direct_utils import is_pre_volta, check_captured_python_definition
 
 
+def require_device_mem_size_gb(required_gb=32.0, didx=0):
```

Comment:
```suggestion
def skip_if_global_memory_below_gb(min_gb: int, gpu_id: int = 0):
```
`skip_if` instead of `require` because that more accurately describes the function. `require` could mean an assertion failure. 

`global` instead of `device` for accuracy. There's also shared memory size. 

`memory` instead of `mem` for https://google.github.io/styleguide/cppguide.html#General_Naming_Rules and consistency since your implementation uses `memory` already.